msf_updates/include/msf_updates/pose_sensor_handler/implementation/pose_sensorhandler.hpp

```diff
@@ -42,6 +43,8 @@ PoseSensorHandler<MEASUREMENT_TYPE, MANAGER_TYPE>::PoseSensorHandler(
             true);
   pnh.param("pose_measurement_world_sensor", measurement_world_sensor_, true);
   pnh.param("pose_use_fixed_covariance", use_fixed_covariance_, false);
+  pnh.param("pose_measurement_minimum_dt", pose_measurement_minimum_dt_, 0.05);
```

Comment:
let's have a default value that's well above the VI-sensor default capture rate?